frontend/src/metabase/dashboard/dashboard.js

```diff
@@ -135,6 +135,24 @@ export const addCardToDashboard = function({ dashId, cardId }: { dashId: DashCar
     };
 }
 
+export const addDashCardToDashboard = function({ dashId, dashcardOverrides }: { dashId: DashCardId, dashcardOverrides: { } }) {
```

Comment:
I've added a dedicated `addDashCardToDashboard` method to be more readable but much of the functionality is shared in `addCardToDashboard`. I can easily overload the `addCardToDashboard` method to handle this extra functionality if that is desired.